### PR TITLE
added CS operations and flows for date/time functionality

### PR DESCRIPTION
**Pull request by cdemergian**
fixes #603

### CODE REVIEW

File: content/io/cloudslang/base/datetime/get_time.sl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+namespace: io.cloudslang.base.datetime
```
Comment (mihaitusa) [inline content/io/cloudslang/base/datetime/get_time.sl]: Description section is missing in all files. Please see the existing content as guidance and fill in with them. Applies to all ...

File: test/io/cloudslang/base/datetime/get_current_time.inputs.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+localeLang: "en"
```
Comment (mihaitusa) [inline test/io/cloudslang/base/datetime/get_current_time.inputs.yaml]: @cdemergian Put here only a placeholder like: localeLang: <localeLang>. Do not hardcode values. Also... as a best practice is better to use snake_case_naming_convention than camelCaseConvention in order to be align with all the content.

File: test/io/cloudslang/base/datetime/test_get_current_time.sl [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#   (c) Copyright 2014 Hewlett-Packard Development Company, L.P.
```
Comment (mihaitusa) [inline test/io/cloudslang/base/datetime/test_get_current_time.sl]: @cdemergian  replace 2014 with 2016 and add Enterprise after HP

Comment (cdemergian) [inline test/io/cloudslang/base/datetime/test_get_current_time.sl]: done

File: content/io/cloudslang/base/datetime/get_time.sl (hunk 1 above)
Comment (cdemergian) [inline content/io/cloudslang/base/datetime/get_time.sl]: done

File: content/io/cloudslang/base/datetime/get_time.sl [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+#!!#
+####################################################
+
+namespace: io.cloudslang.base.datetime
+
+operation:
+  name: get_time
+
+  inputs:
+    - localeLang:
```
Comment (mihaitusa) [inline content/io/cloudslang/base/datetime/get_time.sl]: @cdemergian Correct syntax is: 
- locale_lang 
- localeLang: ${get(locale_lang), "en"}

File: content/io/cloudslang/base/datetime/parse_date.sl [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+####################################################
+
+namespace: io.cloudslang.base.datetime
+
+operation:
+  name: parse_date
+
+  inputs:
+    - date
+    - dateFormat:
```
Comment (mihaitusa) [inline content/io/cloudslang/base/datetime/parse_date.sl]: @cdemergian  add here date_format input with required: false modifier

File: content/io/cloudslang/base/datetime/get_time.sl (hunk 4 above)
Comment (cdemergian) [inline content/io/cloudslang/base/datetime/get_time.sl]: done

File: content/io/cloudslang/base/datetime/parse_date.sl (hunk 5 above)
Comment (cdemergian) [inline content/io/cloudslang/base/datetime/parse_date.sl]: done

File: test/io/cloudslang/base/datetime/offset_time_by.inputs.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+test_offset_time_by:
+  inputs:
+    - date: "April 26, 2016 1:32:20 PM EEST"
+    - offset: "5"
+    - locale_lang: "en"
+    - locale_country: "US"
+  description: simple offset_time_by test
```
Comment (mihaitusa) [inline test/io/cloudslang/base/datetime/offset_time_by.inputs.yaml]: @cdemergian  add here: testFlowPath: io.cloudslang.base.datetime.offset_time_by and retry commit. Applies to all inputs files.

Comment (cdemergian) [inline test/io/cloudslang/base/datetime/offset_time_by.inputs.yaml]: done

**Comment by mihaitusa** [comment]
Wait here for release to MC for cs-date-time module.

**Pull request closed without merging**